Mathlib/CategoryTheory/Shift/CommShift.lean

```diff
@@ -47,6 +47,14 @@ noncomputable def isoZero : shiftFunctor C (0 : A) ⋙ F ≅ F ⋙ shiftFunctor
   isoWhiskerRight (shiftFunctorZero C A) F ≪≫ F.leftUnitor ≪≫
      F.rightUnitor.symm ≪≫ isoWhiskerLeft F (shiftFunctorZero D A).symm
 
+/-- For any functor `F : C ⥤ D` and any `a` in `A` such that `a=0`,
+this is the obvious isomorphism `shiftFunctor C a ⋙ F ≅ F ⋙ shiftFunctor D a` deduced from the
+isomorphisms `shiftFunctorZero'` on both categories `C` and `D`. -/
+@[simps!]
+noncomputable def isoZero' (a : A) (ha : a = 0) : shiftFunctor C a ⋙ F ≅ F ⋙ shiftFunctor D a :=
```

Comment:
There should be a lemma saying `isoZero' _ _ 0 rfl` equals `isoZero`. It would make the proof of `commShiftIso_zero'` easier, because the proof would become something like `subst ha; rw [isoZero'_eq_isoZero, commShiftIso_zero]`.

Reply:
Done.